packages/fxa-profile-server/package.json

```diff
@@ -7,7 +7,7 @@
     "lint": "eslint .",
     "audit": "npm audit --json | audit-filter --nsp-config=.nsprc --audit=-",
     "outdated": "npm outdated --depth 0 || exit 0",
-    "start": "mkdirp var/public && pm2 start pm2.config.js && ../../_scripts/check-url.sh localhost:1111/__heartbeat__",
+    "start": "node scripts/mkdirp.js && pm2 start pm2.config.js && ../../_scripts/check-url.sh localhost:1111/__heartbeat__",
```

Comment:
Not sure if anybody uses Windows, or if this can just change to Bash's `mkdir -p var/public` instead of extracting to a separate script. 🤷

Reply:
This was my first thought also, but I was getting errors building and running tests when I tried using that 😬

Reply:
Glad I checked this -- I have no idea why it was not working previously and is passing now, but it is working now!